### PR TITLE
[es] Spanish folder localization docs/kubernetes #5227

### PR DESCRIPTION
**Pull request by Victorsesan**
I have translated the following page opentelemetry.io/content/en/docs/kubernetes/getting-started.md and created a new page opentelemetry.io/content/es/docs/kubernetes/getting-started.md

Relates to #5227

### CODE REVIEW

**Comment by theletterf** [comment]
Thanks, @Victorsesan! Please add the related issue to the description.

File: content/es/docs/kubernetes/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
+---
+title: Primeros pasos
+weight: 1
+# más bonito-ignorar
```
Comment (theletterf) [inline content/es/docs/kubernetes/getting-started.md:4]:
```suggestion
# más bonito-ignorar
```

Esto no se traduce.

File: content/es/docs/kubernetes/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+cSpell:gnore: filelog filelogreceiver kubeletstats kubeletstatsreceiver loggingexporter otlpexporter sattributes sattributesprocessor sclusterreceiver sobjectsreceiver
+---
+
+Esta página le guiará a través de la forma más rápida de comenzar a monitorear su
+Clúster de Kubernetes usando OpenTelemetry. Se centrará en recopilar métricas y
+registros para clústeres, nodos, pods y contenedores de Kubernetes, además de habilitar
+el clúster para admitir servicios que emiten datos OTLP.
```
Comment (theletterf) [inline content/es/docs/kubernetes/getting-started.md:11]: Por favor, cambia las frases que hablan al usuario de usted a una segunda persona (Esta página te guiará) en todo el documento.

**Review by theletterf — CHANGES_REQUESTED** [review]

**Comment by theletterf** [comment]
@Victorsesan Añade hashes de los originales según https://github.com/open-telemetry/opentelemetry.io/actions/runs/11184480209/job/31121250608?pr=5301

Para instrucciones, mira https://opentelemetry.io/docs/contributing/localization/#the-default_lang_commit-front-matter-field

**Comment by theletterf** [comment]
@Victorsesan Seems like the issues persist. Let me know if you need assistance or comment in the channel.

**Comment by Victorsesan** [comment]
@theletterf Sí, gracias por preguntar, lo siento, no quería ser un problema para hacer demasiadas preguntas para un problema básico como este. El primer problema que tuve fue encontrar la clave hash. El enlace solo describía cómo agregar default_lang_commit pero no mencionaba cómo encontrarlo, así que usé git log -1 --format="%H para encontrar la última clave hash de commit y la usé como default_lang_commit en cada archivo que no estoy seguro de que fuera el método correcto para obtener la clave hash, así que cualquier ayuda en esta área sobre cómo encontrar la clave hash será apreciada. Además, el registro de errores de escritura se refiere a líneas incorrectas, lo que dificulta saber qué línea se encuentra exactamente el error de escritura. Gracias

**Comment by theletterf** [comment]
@krol3 Could you help Victor? Thanks!

**Comment by Victorsesan** [comment]
@theletterf  Had made some translations and opened a PR , will be awaiting expected test results for alteration👍

**Comment by theletterf** [comment]
@Victorsesan I see some fields in the frontmatter translated to Spanish, like `linkTítulo` -- please don't translate frontmatter setting names. Also, check the hashes, as they might differ for each page.

**Comment by Victorsesan** [comment]
> @Victorsesan I see some fields in the frontmatter translated to Spanish, like `linkTítulo` -- please don't translate frontmatter setting names. Also, check the hashes, as they might differ for each page.

Thanks for reviewing, i have had them sorted. Also i have been wondering are the methods to check default language commit in the [Contribution.md](https://opentelemetry.io/docs/contributing/localization/#the-default_lang_commit-front-matter-field) really up to date 🤔, maybe is just me or i haven't been doing it right but each time i run something like ```npm run check:i18n -- -n``` i always get this error of 'scripts' not recognized as an internal or external command, operable program or batch file, and thesame even while trying the other commands. 

So far i have been using this rather ```git log -1 --format="%H"``` which provides a hash but I'm not sure if it works thesame as that which was written in the contribution.md, and unfortunately i couldn't get the hash of each page with it. Hoping to get a better understanding on how this work so i can have everything sorted out as  requested thanks

**Comment by theletterf** [comment]
@Victorsesan I'm closing this PR. It has too many faults to fix, including automated translations that weren't checked by a human, such as "coleccionista" instead of "Collector". This is not the right way of localizing content.

Please, try again with fewer files and following all the recommendations in

**Pull request closed without merging**